docs/resources/permission_set.md

```diff
@@ -0,0 +1,31 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "looker_permission_set Resource - terraform-provider-looker"
+subcategory: ""
+description: |-
+  Manages a Looker permission set
+---
+
+# looker_permission_set (Resource)
+
+Manages a Looker permission set
+
```

Comment:
To keep consistent with other docs, you could add a 

`Example Usage` section here with the example you have in examples/resources/permission_set/main.tf :)

Reply:
good point, not sure why the generator missed it unless it's supposed to be manual?

Reply:
I've been doing it manually :'( there's a ticket to configure the generator to massively improve the dev experience!!